Assets/MixedRealityToolkit.Examples/Demos/UX/BoundingBox/Scripts/BoundingBoxTest.cs

```diff
@@ -192,20 +192,45 @@ private IEnumerator Sequence()
 
                 GameObject multiRoot = new GameObject();
                 multiRoot.name = "multiRoot";
-                multiRoot.transform.position = cubePosition;
+                Vector3 forwardOffset = Vector3.forward * .5f;
```

Comment:
Can you actually rename this to BoundingBoxExampleTest or BoundingBoxExample? There is both a BoundingBoxTest.cs and a BoundingBoxTests.cs which is very confusing. The latter though I believe is the only one that goes through CI and our PR automation checking.

Also, I'm still going through all of this PR. But should there be any tests modified or added in BoundingBoxTests.cs for this change?